src/js/main.js

```diff
@@ -23,7 +21,23 @@
     })//END OF RESULTS
 
 
-
   })//END OF MODULE
+  .controller('Find', ['$http', '$scope', function($http, $scope){
+    var BASEURL = 'http://wishcastr-staging.herokuapp.com/products/';
+
+    $scope.query = "";
+    $scope.products = { };
+    console.log($scope.query);
+    $scope.search = function(){
+      $http.get(BASEURL+'search.json?query='+$scope.query)
+      .then(function(response){
+        $scope.products = response.data;
```

Comment:
Same in here, although this function should fire when the HTTP request is successful.